spec/smartcar/unit/utils_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Temp
+  include Smartcar::Utils
+end
+RSpec.describe Smartcar::Utils do
+  subject do
+    Temp.new
+  end
+
+  describe 'get_config' do
```

Comment:
Would you want a test for the logic for changing the name when `mode == 'TEST'`?

Reply:
I guess these are legacy tests, but i will add an explicit test for that too. In short this file is tested as a part of various other tests (test suite wouldnt work if that doesnt work.)